src/runtime/ready.dev.ts

```diff
@@ -1,9 +1,9 @@
-import { defineNitroPlugin } from 'nitropack/runtime'
-import { hubHooks } from './server/utils/hooks'
+import { defineNitroPlugin } from 'nitropack/runtime/plugin'
```

Comment:
This should be `nitropack/runtime` (`runtime/` subpaths are being deprecated)

Reply:
Updated, thanks for looking into it @pi0 💚